packages/neon_framework/pubspec.yaml

```diff
@@ -50,9 +50,11 @@ dependencies:
       url: https://github.com/nextcloud/neon
       path: packages/sort_box
   sqflite: ^2.3.0
-  sqflite_common_ffi: ^2.2.8-2
+  sqflite_common_ffi: ^2.3.2
+  sqflite_common_ffi_web: ^0.4.2+3
```

Comment:
We shouldn't need web right now as we don't use the cache there. @Leptopoda should we keep this in light of https://github.com/nextcloud/neon/issues/1538?